source/chapters/library.rst

```diff
@@ -866,6 +867,86 @@ different location. Select :guilabel:`Use Default Library` to reset.
   Mac partition mounted in Linux, you can load it and use your iTunes tracks and
   playlists as well.
 
+.. _import-serato-metadata:
+
+Importing metadata from Serato
+------------------------------
+
+From Mixxx 2.3, tracks in your Serato library and crates can now be imported into Mixxx library table.
+Tracks can also be loaded into deck without needing to add the music directory in the preferences. 
+You can also import your Serato library into Mixxx that are in USB drives for a portable Serato library.
+
+**How to import Serato libraries from external USB drive**
+
+* Copy music library from Serato into external USB drive
+* Plug in USB drive while Mixxx is running
+* Click on Serato in the library, all records on USB from Serato will be visible (attach as many as you like!)
+* Then click on the desired removable device, and all your folders and playlists from Serato will be revealed for you to play from
+
+Import your Beatgrid and Hotcues
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Track cue points are saved in special file tags as well as the track metadata. 
+You can now import your track cue points and all of it's metadata from Serato to Mixxx.
+
+Once you import your Serato library into Mixxx and load a track for the first time, the positions, labels and colors of your hotcues, other metadatas and beatgrid are automatically imported.
+
+If you have already added a track to your Mixxx library before support for reading Serato's hotcues was added, you can reimport metadata from Serato via the track context menu.
+This action will clear your existing cuepoints in Mixxx if the track has any Serato hotcues.
+
+**To reimport metadata for files**
+
+* Right click on the track and select metadata
+* Click on Import from file tag. If the track has Serato Metadata, all existing cue points will be overwritten
+
+Decodes may slightly detect different track start and end times for files from different sources, this could possibly be because of the presence of countless encoders and decoders for MP3 and M4A/AAC.
```

Comment:
```suggestion
Decoders may slightly detect different track start and end times for files from different sources, this could possibly be because of the presence of countless encoders and decoders for MP3 and M4A/AAC.
```